### PR TITLE
Events list design updates

### PR DESCRIPTION
**Pull request by davidrapson**
Events list design updates. Also cleans up the event item fragments a bit, by splitting out different variants (rather than passing lots of flags into one fragment)

# Events - Large
events list - large

# Events - Medium
events list - medium

# Events - Small
events list - small

# Masterclasses

masterclasses list

### CODE REVIEW

**Comment by davidrapson** [comment]
Smallest yet… @wpf500 

screen shot 2015-01-08 at 17 47 01

**Comment by davidrapson** [comment]
Made a few minor design refinements from review, but otherwise got the OK from Ben, so ready for review.

File: frontend/assets/stylesheets/components/_action.scss [hunk 1]
```diff
@@ -62,6 +62,7 @@
 // TODO: Specificy issues
 // this variant moves the arrow to the other side
 // and hides the one on the right
+// Onlt used on change tier  / downgrade
```
Comment (mattandrews) [inline frontend/assets/stylesheets/components/_action.scss]: This is the most nitpicky thing ever but typo in "Onlt"

File: frontend/assets/stylesheets/components/_header-bar.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   ========================================================================== */
+
+.header-bar {
+    @include clearfix;
+    position: relative;
+    padding-left: rem($gs-gutter / 2);
+    padding-right: rem($gs-gutter / 2);
+
+    @include mq(tablet) {
+        // min-height: rem(132px);
```
Comment (mattandrews) [inline frontend/assets/stylesheets/components/_header-bar.scss]: Can probably remove this

File: frontend/assets/stylesheets/components/_header-bar.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        padding-left: rem($gs-gutter);
+    }
+
+    @include mq(mem-full) {
+        padding-left: rem(gs-span(2) + $gs-gutter * 2); // 180
```
Comment (mattandrews) [inline frontend/assets/stylesheets/components/_header-bar.scss]: :one: :eight: :zero: seems a bit pointlesss

**Comment by mattandrews** [comment]
LGTM besides some absolutely minor/trivial asides. :+1:

**Comment by davidrapson** [comment]
@jamesoram Mind giving this the once over?

**Comment by jamesoram** [comment]
:+1: it looks pretty good

**Pull request merged**